docs/concepts/registries.md

```diff
@@ -62,7 +62,7 @@ class AnnotatedHolder {
                                                     // Object to be injected: "neomagicae:coffeinum" from the ManaType registry.
 
     public static final Item ENDER_PEARL = null;    // Note that the actual name is "minecraft:ender_pearl", not "minecraft:ENDER_PEARL".
-                                                    // Therefore, THIS WILL FAIL. The field has to be lowercased or annotated.
+                                                    // However, since constructing a ResourceLocation lowercases the value, this will work.
```

Comment:
I didn't realize this until just a few days ago. I think @PaleoCrafter was the one who pointed it out to me